frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -177,6 +187,9 @@ local OBSOLETED_CSS = {
 
 function ReaderTypeset:onSetRenderDPI(dpi)
     self:setRenderDPI(dpi)
+    UIManager:show(Notification:new{
+        text = T( _("Zoom set to %1 dpi."), dpi),
```

Comment:
That's a bit janky, but that's not your fault, the whole thing is ;p.

(I mean, mixing zoom and dpi makes no sense, but it's called dpi, and it does sort of behaves like a zoom :/).